src/Microsoft.Security.Utilities.Core/AzureSearchIdentifiableKeys.cs

```diff
@@ -1,37 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-
-namespace Microsoft.Security.Utilities;
-
-internal sealed class AzureSearchIdentifiableKeys : RegexPattern
```

Comment:
>AzureSearchIdentifiableKeys [](http://example.com/codeflow?start=22&length=27)

This internal sealed class is entirely unused. It's a vestige of a base class we never used for the admin and query search rules.